exec/java-exec/src/main/java/org/apache/drill/exec/planner/sql/handlers/DefaultSqlHandler.java

```diff
@@ -162,6 +166,9 @@ public PhysicalPlan getPlan(SqlNode sqlNode) throws ValidationException, RelConv
 
     log("Optiq Logical", queryRelNode, logger);
     DrillRel drel = convertToDrel(queryRelNode, validatedRowType);
+    if (otherPlan != null) {
```

Comment:
The DrillRel node for pojo record reader needs to have corresponding Prel and Prule, right?